uniffi_bindgen/src/interface/mod.rs

```diff
@@ -219,6 +219,25 @@ impl ComponentInterface {
         self.errors.iter().find(|e| e.name == name)
     }
 
+    /// Should we generate read/lower functions for errors?
+    ///
+    /// This is a temporary measure since lower/write can't be generated for some errors,
```

Comment:
This is still slightly confusing to me (which is a comment on me, not on this patch :) But in what way is this temporary?

Reply:
I think that was optimistic on my part.  Once we implement `#1348`, then we can tell users that they shouldn't use the `enum` style definitions for errors if their errors actually have fields, and at some point we can drop support for it.  Maybe a simpler road would be to fully implement proc-macros and drop support for UDL.  But both of those things seems far off so I changed the wording.